### DISCUSSION TITLE
Verbose errors when stuff aren't working as expected

### DISCUSSION THREAD

**Discussion by emilhem** [stern/stern#138] [Q&A] [answered]
I'm trying to fault trace why the stern command isn't doing expected stuff. Is there a way to turn on verbose logging?

I've tried `--verbose` and `-V`.

Use case:
The issue I'm trying to investigate is receiving `Error: No matching labels`.

**Comment by superbrothers** [comment]
> I've tried --verbose and -V.

stern has no verbose flag. Could you create an issue about it?

> The issue I'm trying to investigate is receiving Error: No matching labels.

`-p`, `--prompt` selects Pods with 'app.kubernetes.io/instance' label. Perhaps there are no Pods with that label.

This error message is difficult to understand, so I would like to change it as follows:
```
No matching labels with 'app.kubernetes.io/instance'
```

**Reply by emilhem** [reply]
I created the issue #139 now :)

I tried using the prompt option but it didn't help unfortunately.

**Comment by superbrothers** [comment, marked as answer]
`--verbosity` flag has been added since stern v1.23.0. It can be used as follows:

```
stern . --verbosity=6
```

See https://github.com/stern/stern#log-level-verbosity for more details.
